test/support/file_endpoint.ex

```diff
@@ -73,12 +68,15 @@ defmodule Membrane.RTC.Engine.Support.FileEndpoint do
       clock_rate: state.track.clock_rate
     }
 
-    spec =
+    parser = parser_interceptor(state.track.encoding)
```

Comment:
Not sure if parser_interceptor is the most fortunate name as we also return encoder/decoder. If you don't find anything better we can keep it